viral/learning_stages.py

```
@@ -1,133 +1,346 @@
+        session = Cached2pSession.model_validate_json(f.read())
+    spks_path = TIFF_UMBRELLA / session.date / session.mouse_name / "suite2p" / "plane0"
+
+    assert (
+        spks_path / "full_grosmark_oasis_preprocessed.npy"
+    ).exists(), (
+        f"File {spks_path / 'full_grosmark_oasis_preprocessed.npy'} does not exist"
+    )
+    spks = np.load(spks_path / "oasis_spikes.npy")
+
+    for rewarded in [False, True]:
+        pcs_mask, smoothed_matrix, place_threshold = get_place_cells(
```

Ruff (0.14.0)

Unpacked variable `pcs_mask` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)